src/components/create/SendBitcoin.vue

```diff
@@ -218,15 +202,23 @@ export default class SendBitcoinLedger extends Vue {
     return currentComponent;
   }
 
+  @Emit()
+  setTxBuilder():void {
```

Comment:
Same question here... why do we need wallet specific txBuilders ? I insist on reconsidering changing the definition of txBuilder as this seems a bad practice

Reply:
I'm going to change the txBuilder to be accesed from the store